packages/manager/modules/sms/src/sms/options/smppParameter/smppParameter.controller.js

```diff
@@ -1,19 +1,46 @@
 export default class SmsSmppParameterCtrl {
   /* @ngInject */
-  constructor(SmsService) {
+  constructor(SmsService, $timeout, $state) {
     this.smsService = SmsService;
+    this.$timeout = $timeout;
+    this.$state = $state;
   }
 
   $onInit() {
+    this.timer = null;
+    this.checkStatus = this.checkStatus.bind(this);
```

Comment:
This line could be removed safely
https://github.com/ovh/manager/pull/8784/files#diff-38f929e0b6174bcb765649cbb60135040a628b126368975cf87472a314186a22R27
When you call `$ctrl.checkStatus()`, `checkStatus`'s context is always `$ctrl`